src/main/webapp/mentors.js

```diff
@@ -14,6 +15,37 @@ function initForm() {
   return;
 }
 
+/**
+ * Poopulates mentor info for the mentor breakout pages.
+ */
+function populateMentorInfo() {
+  const mentorContainer = document.getElementById('mentor-breakout-container');
+  if (!mentorContainer) {
+    return;
+  }
+  const curUrl = new URL(window.location.href);
+  const userID = curUrl.searchParams.get('mentorID');
+  console.log(userID);
+  const url = new URL('/mentor_breakout', window.location.origin);
+  url.searchParams.append('mentorID', userID);
+  const fetchRequest = standardizeFetchErrors(
+      fetch(url),
+      'Failed to communicate with the server, please try again later.',
+      'Encountered a server error, please try again later.',
```

Comment:
I copied these error messages from your servlet so that server errors would always be handled the same way. Thank you for letting me know that you changed them.